### PR TITLE
updated stack to lts-18.10

### PR DESCRIPTION
**Pull request by piq9117**
Following changes are related to stack lts-18.10

### CODE REVIEW

File: test/integration/TestImport.hs [hunk 1]
```diff
@@ -85,7 +87,7 @@ wipeDB :: App -> IO ()
 wipeDB app = runDBWithApp app $ do
   tables <- getTables
   sqlBackend <- ask
-  let escapedTables = fmap ( connEscapeName sqlBackend . DBName ) tables
+  let escapedTables = fmap ( connEscapeFieldName sqlBackend . FieldNameDB ) tables
```
Comment (lowderdev) [inline test/integration/TestImport.hs:90]: I was running into this after updating persistent! Nice to know the fix!

**Comment by lowderdev** [comment]
The integ tests fail for me. 🤔   Did I miss a setup step perhaps?
```
real-world-yesod on  update-to-lts-18.10 ➜ stack test :integration
real-world-yesod> test (suite: integration)


Handler.Profile
  ProfileSpec
    asserts no access to my-account for anonymous users FAILED [1]
    will return a guest with an authenticated request FAILED [2]

Failures:

  test/integration/Handler/ProfileSpec.hs:13:5:
  1) Handler.Profile.ProfileSpec asserts no access to my-account for anonymous users
       uncaught exception: IOException of type OtherError
       libpq: failed (FATAL:  password authentication failed for user "real-world-yesod-test"
       )

  To rerun use: --match "/Handler.Profile/ProfileSpec/asserts no access to my-account for anonymous users/"

  test/integration/Handler/ProfileSpec.hs:17:5:
  2) Handler.Profile.ProfileSpec will return a guest with an authenticated request
       uncaught exception: IOException of type OtherError
       libpq: failed (FATAL:  password authentication failed for user "real-world-yesod-test"
       )

  To rerun use: --match "/Handler.Profile/ProfileSpec/will return a guest with an authenticated request/"

Randomized with seed 957180923
```

**Comment by piq9117** [comment]
thanks for trying it out! It looks like it's not able to communicate to a test postgresql db. Try running the test db with 
```
docker-compose up db-test
```
then run the migration with
```
docker-compose up flyway-test
```
Then run the tests with `stack test`. Hopefully this will succeed :crossed_fingers:

**Comment by lowderdev** [comment]
Huh... I thought I had followed those steps the first time, but maybe I had accidentally done `docker-compose up db-dev` or something because they're passing now!

**Review by lowderdev — APPROVED** [review]
Tests passing for me!

**Pull request merged**